src/vivarium_public_health/risks/distributions.py

```diff
@@ -28,6 +28,10 @@ def _get_parameters(self, weights, mean, sd):
 
     def ppf(self, q):
         if not q.empty:
+            # We limit valid propensity to (0.001 0.999). Beyond that bound values return NaN and then become zero,
+            # which is nonsensical.
+            q[q > 0.999] = 0.998
```

Comment:
can propensity be 0.999? Your comment makes it seem like the range is exclusive, but then only bump down outside of the end points (making it seem inclusive)?

Reply:
I'm unsure, because I can't actually see where the limit spawns from, but I bumped inside the limit to be safe, and I made the comment show exclusive to be safe as well

Reply:
I'll test it

Reply:
It's appears to be inclusive. I think the limit comes from a percentage of a lognormal distribution. I think not brushing against the limit is safer, though, and is what james and I discussed

Reply:
Ok I guess I'm just confused over whether or not a propensity of 0.999 (which won't be adjusted down to 0.998 but from the comment seems to be invalid) will result in the NaN -> 0 issue

Reply:
Oh nvm! Just saw the updated comment.

Reply:
yeah that's fair. it's weird to leave 999 and resassign greater than 999 to 998. I'll just make it inclusive.